packages/semantic-ui/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -68,12 +68,12 @@ export default function FieldTemplate<
       <Form.Group key={id} widths='equal' grouped>
         <MaybeWrap wrap={wrapContent} className='sui-field-content'>
           {children}
-          {displayLabel && rawDescription && (
+          {displayLabel && description && (
```

Comment:
`rawDescription` should continue to be used as the condition since it will be null when there is no description provided. This should cause most of the need to update the snapshots to go away when the description is an empty React element.
```suggestion
          {displayLabel && rawDescription && (
```

Reply:
Got it